.h5pignore

```diff
@@ -12,3 +14,6 @@ storybook-static
 docs
 babel.config.js
 tsconfig.json
+LICENSE
```

Comment:
LICENSE should be part of the h5p package

```suggestion
```

Reply:
Sorry, LICENSE has to be excluded as well. H5P core will not allow files without any file extension.

If you want to keep the license information in the H5P library package, then you should rename it to LICENSE.txt which will also be identified by github automatically and would be allowed inside H5P library packages.